roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_from_catalog.yml

```diff
@@ -26,13 +26,23 @@
       > {{ artifact_extra_logs_dir }}/gpu_operator_packagemanifest.json
 
   rescue:
+  - name: Capture the Catalog Operator logs (debug)
+    shell:
+      oc logs deployment.apps/catalog-operator
```

Comment:
Why do we even collect these logs, aren't they available in `gather-extra` anyway?

Reply:
they are ... as long as we're running in Prow CI

and from my point of view/experience, it's the most efficient way to troubleshot when then likely cause of outage are stored along with the test results. The `(debug)` tag in the name informally says that this task isn't part of the test but only there for debugging/troubleshooting/observability

Reply:
> they are ... as long as we're running in Prow CI
Fair enough

> it's the most efficient way to troubleshot when then likely cause of outage are stored along with the test results

With the existence of things like https://github.com/kxr/o-must-gather, I must say I disagree - it's easier for me to find things with `omg` on the Prow must-gather than having to "know" about specialized artifacts that we create

Reply:
if the test fail, how do you (an external observer) know where to go for troubleshooting the issue?

as the role implementor, I give you the most likely culprint

ideally (my ideal at least), for every known `flake`, you wouldn't have to go to `gather-extra` to know what issue caused the flake

Reply:
Yeah that's a good point